core/common/src/main/java/alluxio/util/FormatUtils.java

```diff
@@ -211,7 +211,7 @@ public static String formatPermission(short permission, boolean isDirectory) {
    * @return the JSON-encoded version of the input string
```

Comment:
@aaudiber I have decided to rework this, creating a RestUtils.createResponse() utility, which handles the special case for strings the same way the TestCase does.